addon/ActivityStreams.js

```diff
@@ -113,7 +113,7 @@ ActivityStreams.prototype = {
     this._initializePerfMeter();
     this._initializeAppURLHider();
 
-    if (!this.options.shield_variant) {
+    if (!this.options.shield_variant && simplePrefs.prefs.activateExperiments) {
```

Comment:
I suppose if this worked for shield, it might be okay here. In particular, I thought we discussed yesterday about an approach that forces each experiment's pref to be set to the `control` value. I believe we happen to be okay right now because all the control values are `false`, so any pref check happens to get a `false`-y value.

Probably to avoid accidental breakage of non-`false` control values, we should shift where we "disable" things.